src/main/kotlin/app/revanced/patches/music/ad/video/fingerprints/ShowMusicVideoAdsConstructorFingerprint.kt

```diff
@@ -0,0 +1,39 @@
+package app.revanced.patches.music.ad.video.fingerprints
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import app.revanced.patcher.fingerprint.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.fingerprint.method.annotation.MatchingMethod
+import app.revanced.patches.music.ad.video.annotations.MusicVideoAdsCompatibility
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+@Name("show-video-ads-constructor-fingerprint")
+@MatchingMethod(
+    "Laghd",
```

Comment:
```suggestion
    "Laghd;",
```

Reply:
No need for the setVisibility anymore. The patch became even simpler and more close to what the Vanced team did 😄 
Style suggestion fixed.
Are you able to merge it now?